When build in a docker container chkshsgr fails:
| ./chkshsgr || ( cat warn-shsgr; exit 1 )
| Oops. Your getgroups() returned 0, and setgroups() failed; this means
| that I can't reliably do my shsgr test. Please either ``make'' as root
| or ``make'' while you're in one or more supplementary groups.
| make: *** [hasshsgr.h] Error 1
So, simply remove this test.

diff -up runit-2.1.2/src/Makefile.docker runit-2.1.2/src/Makefile
--- runit-2.1.2/src/Makefile.docker	2019-10-11 13:40:09.231258709 +0200
+++ runit-2.1.2/src/Makefile	2019-10-11 13:40:39.845194744 +0200
@@ -239,7 +239,6 @@ hassgprm.h: choose compile hassgprm.h1 h
 
 hasshsgr.h: chkshsgr choose compile hasshsgr.h1 hasshsgr.h2 load \
 tryshsgr.c warn-shsgr
-	./chkshsgr || ( cat warn-shsgr; exit 1 )
 	./choose clr tryshsgr hasshsgr.h1 hasshsgr.h2 > hasshsgr.h
 
 haswaitp.h: choose compile haswaitp.h1 haswaitp.h2 load trywaitp.c
